swan-lake/learn/quick-tour.md

```diff
@@ -12,14 +12,62 @@ redirect_from:
 
 ## Installing Ballerina
 
-1. [Download](https://ballerina.io/downloads) Ballerina based on the Operating System you are using. 
-1. Follow the instructions given on the [Getting Started](/swan-lake/learn/getting-started) page to set it up. 
-1. Follow the instructions given on the [Visual Studio Code Plugin](/swan-lake/learn/tools-ides/vscode-plugin) page or the [IntelliJ IDEA Plugin](/swan-lake/learn/tools-ides/intellij-plugin) page to set up your preferred editor for Ballerina.
+1. [Download](/downloads) Ballerina based on the Operating System you are using. 
+1. [Install](/swan-lake/learn/installing-ballerina) Ballerina. 
+1. Set up [Visual Studio Code](/swan-lake/learn/tools-ides/vscode-plugin) for Ballerina.
 
-## Writing a Service, Running It, and Invoking It
+## Creating a Client to Invoke a Service 
```

Comment:
What is the reason for moving the `Client` example to the top? IMO, services are one of the key interesting feature in ballerina. So shouldn't we start with service first?

Reply:
Brought "writing the service" part back to the top.